third_party/move/tools/move-asm/tests/assembler/friends.masm

```diff
@@ -0,0 +1,9 @@
+//# print-bytecode --input module
+
+// Notice we cannot actually test friend publishing since the test framework doesn't support module bundles,
```

Comment:
@georgemitenkov made some changes recently which I think allows testing friends (I think the sequence is: publish module A, publish module B which mentions A as a friend, republish module A to use module B's friend function).

We could add a TODO here instead if we don't want to do this in this PR.